frameworks/digital-outcomes-and-specialists/questions/briefs/essentialRequirementsOutcomes.yml

```diff
@@ -1,17 +1,18 @@
-name: Essential skills and experience
+name: 'Technical competence: essential skills and experience'
```

Comment:
This `name` is going to look really long in the summary table.  The reason for splitting into `name` and `question` was for exactly this reason - the `name` is a short version to appear in summary tables.

See under `name` here for details: https://github.com/alphagov/digitalmarketplace-frameworks/blob/master/README.md